packages/xstate-graph/rollup.config.js

```diff
@@ -0,0 +1,65 @@
+import typescript from 'rollup-plugin-typescript2';
+import { terser } from 'rollup-plugin-terser';
+import rollupReplace from 'rollup-plugin-replace';
+import fileSize from 'rollup-plugin-filesize';
+
+const createTsPlugin = ({ declaration = true, target } = {}) =>
+  typescript({
+    clean: true,
+    tsconfigOverride: {
+      compilerOptions: {
+        declaration,
+        ...(target && { target })
+      }
+    }
+  });
+
+const createNpmConfig = ({ input, output }) => ({
+  input,
+  output,
+  preserveModules: true,
+  plugins: [createTsPlugin()]
+});
+
+const createUmdConfig = ({ input, output, target = undefined }) => ({
+  input,
+  output,
+  plugins: [
+    rollupReplace({
+      'process.env.NODE_ENV': JSON.stringify('production')
+    }),
+    createTsPlugin({ declaration: false, target }),
+    terser({
+      toplevel: true
+    }),
+    fileSize()
+  ]
+});
+
+export default [
+  createNpmConfig({
+    input: 'src/index.ts',
+    output: [
+      {
+        dir: 'es',
+        format: 'esm'
+      }
+    ]
+  }),
+  createUmdConfig({
+    input: 'src/index.ts',
+    output: {
+      file: 'dist/xstate-graph.js',
+      format: 'umd',
+      name: 'XStateGraph'
+    }
+  }),
+  createUmdConfig({
```

Comment:
let's remove those `.web` builds for now - only `xstate` has it and I'm still unsure how people actually utilize it, or rather - how they would utilize a build like this which has actual external dependencies (xstate has no dependencies, but this one depends on xstate)